.github/workflows/build-and-push-containers.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       DOCKER_USER: ${{ secrets.CONTAINER_USERNAME }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

Comment:
very weird for it to pick a hash here

Reply:
v4 is working fine on rector, see PR:

- https://github.com/rectorphp/rector-src/pull/4909

this reported at:

- https://github.com/actions/checkout/issues/1448

Reply:
got fixed